app/config/security.php

```diff
@@ -88,6 +88,7 @@
                 'bazinga_oauth'      => true,
                 'mautic_plugin_auth' => true,
                 'stateless'          => true,
+                'http_basic'         => true,
```

Comment:
Change `true` here with `%mautic.api_enable_basic_auth%` so that it honors the config setting.